buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```
@@ -24,7 +24,7 @@ java {
 
 checkstyle {
   configDirectory.set(file("$rootDir/buildscripts/"))
-  toolVersion = "8.12"
+  toolVersion = "8.45.1"
```

Maybe we just try the much newer version, like what we're using in the core java repo?

```suggestion
  toolVersion = "10.20.1"
```